common/beerocks/bcl/source/network/network_utils.cpp

```diff
@@ -898,7 +898,16 @@ bool network_utils::arp_send(const std::string &iface, const std::string &dst_ip
     struct sockaddr_ll sock;
     uint8_t packet_buffer[128];
 
-    uint32_t dst_ip_uint = network_utils::uint_ipv4_from_string(dst_ip);
+    // If the destination IP is empty, there is no point sending the arp, therefore replace the 
+    // with broadcast IP, so all clients will receive it and answer, but since the request is
+    // being sent to a specific mac address, then only the requested client will answer.
+    uint32_t dst_ip_uint; 
+    if (dst_ip.empty()) {
+        dst_ip_uint = network_utils::uint_ipv4_from_string("255.255.255.255");
```

Comment:
```suggestion
        dst_ip_uint = 0xFFFFFFFF; // Broadcast IP address
```

Reply:
We're using a broadcast address (it's not wildcard). Please update the commit message.